src/MongoDB.Driver/MongoCollectionImpl.cs

```diff
@@ -746,6 +812,14 @@ private DistinctOperation<TField> CreateDistinctOperation<TField>(FieldDefinitio
             };
         }
 
+        private CountOperation CreateEstimatedDocumentCountOperation(EstimatedDocumentCountOptions options)
+        {
+            return new CountOperation(_collectionNamespace, _messageEncoderSettings)
```

Comment:
At first I thought we might need a new operation for EstimatedDocumentCount but the spec says EstimatedDocumentCount is basically the original Count just without a filter.